content/en/monitors/status/status_page.md

```diff
@@ -0,0 +1,100 @@
+- Investigate the possible cause of an alert
+- Take action to escalate, maintain, or resolve your investigation
+
+## Header
+
+{{< img src="/monitors/status/status_page_header.png" alt="Monitor status page header" style="width:100%;" >}}
+
+The header contains the monitor's metadata including:- Alert status
+- Monitor status
+- Monitor type
+- Creation date
+- Author
+- Associated teams (if available)
+- Tags
+- Monitor quality actions
+
+On the right, you'll find the **Edit**, **Clone**, **Export**, and **Permissions** buttons.
+
+From the header, you can resolve the monitor (resolving all groups in alerts) and manage downtimes. For more information, see [Resolve an alert](#resolve-an-alert). 
+
+Additionally, you can view downtimes impacting this monitor and create new ones to mute notifications.
+
+## View monitor evaluations through graphs
+
+ {{< img src="/monitors/status/view_monitor_evaluations_graphs.png" alt="Example monitor evaluation graph, filtered by the prod account template variable." style="width:100%;" >}}
+
+You can troubleshoot monitor alerts by viewing your data through graphs. Toggle between the **Evaluated Data**, **Source Data**, and **In Groups** graphs to investigate which groups are causing the alert. The graphing component also contains details on the monitor query, configured monitor evaluation, and notification information. To learn more about each graph, see the [Monitor status graphs][1] documentation.
+
+Use the template variables to scope down the monitor page to specific groups, select the attributes you want to filter by. 
+
+## View monitor events details
+
+{{< img src="/monitors/status/status_page_event_details.png" alt="Example monitor status event details" style="width:100%;" >}}
+
+You can explore different product areas while maintaining the same alerting context, ensuring you view the same timeframe and service parameters as on your monitor's status page. Use Event details to troubleshoot if the monitor is alerting due to recent configuration changes, take the query to other product areas to troubleshoot and cross reference without losing context. 
+
+For more information, see the [Monitor status events][2] documentation.
+
+### Resolve an alert
+
+You can resolve a monitor alert from the Header or the Next Actions sections in the Event details. Resolving from the Header sets the entire monitor to 'OK' (all groups), while resolving from Next Actions only affects the group related to the selected event.
+
+From Next Actions, click the **Resolve** button to temporarily set the monitor status to 'OK' for its next evaluation. This does not acknowledge the alert or instruct Datadog to ignore it. Use Resolve when data is reported intermittently, such as when a monitor triggers an alert but stops receiving data, preventing it from recovering to 'OK' on its own. The 'resolve' function or the 'Automatically resolve monitor after X hours' option resets the monitor to 'OK'. 
```

Comment:
> From Next Actions, click the **Resolve** button to temporarily set the monitor status to 'OK' for its next evaluation.

I got a bit lost when I read this. when the user click on resolve in the next actions (within the event details) only the group is resolved, so it's not sure that the monitor status will be set to ok since there might be other groups in alert and therefore the monitor status will stay in alert.

Reply:
Maybe we should use what we had [before](https://docs.datadoghq.com/monitors/status/#resolve). At the end nothing changed on the functionality, it works at the same way